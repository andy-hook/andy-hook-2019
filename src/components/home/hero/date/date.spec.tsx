import React from "react"
import { shallowWithTheme } from "@test-utils"

import Date from "./date"

describe("<Date />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme("dark", <Date />)
    expect(tree).toMatchSnapshot()
  })
})
