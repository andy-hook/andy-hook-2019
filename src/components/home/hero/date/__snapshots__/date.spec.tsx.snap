// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Date /> renders correctly 1`] = `ShallowWrapper {}`;
